#hand_command.msg

string hand_name
int32[] finger_ids
int32[] angles
int32[] forces
int32[] speeds
